.github/ISSUE_TEMPLATE/page-report-ru.yml

```
@@ -5,58 +5,58 @@ body:
   - type: markdown
     attributes:
       value: |
-        ### Before you start
+        ### Прежде чем начать
 
-        **Want to change this page yourself?** This content is open source!
-        ↩ Go back and use the _Edit on GitHub_ link on the page.
+        **Готовы внести правки самостоятельно?** Справочник MDN может редактировать каждый!
+        ↩ Для этого вернитесь обратно в статью и кликните на ссылку _Edit on GitHub_.
 
-        **Is your issue about the browser compatibility table?**
-        ↩ Go back and use the _Report problems with this compatibility data on GitHub_ link on the page.
+        **В таблице с браузерной совместимостью ошибка?**
+        ↩ Такие ошибки правит отдельная команда. Пожалуйста, вернитесь обратно в статью и кликните на ссылку _Report problems with this compatibility data on GitHub_, чтобы сообщить им.
 
-        **Need help with a browser?**
-        🙋 To get help with [Firefox](https://support.mozilla.org/en-US/kb/file-bug-report-or-feature-request-mozilla), [Chrome](https://support.google.com/chrome/answer/95315?hl=en-GB&ref_topic=7439544), [Safari](https://www.apple.com/feedback/safari.html), or another browser, check the browser's support site.
+        **Технология из статьи не работает в вашем браузере?**
```

Не знаю насколько такая правка будет релевантна. Но я бы так перевёл:
```suggestion
        **Что-то из статьи не работает в вашем браузере?**
```